composer.json

```diff
@@ -24,7 +24,7 @@
         "php" : ">=5.4",
         "squizlabs/php_codesniffer" : "^3.13.3 || ^4.0.0",
         "phpcompatibility/php-compatibility" : "^10.0.0@dev",
-        "dealerdirect/phpcodesniffer-composer-installer" : "^0.7 || ^1.0"
```

Comment:
Note: we inherit the Composer installer plugin now via both PHPCSExtra and PHPCompatibility.
Better not to have our own requirement to prevent conflicts between the allowed versions.